bpdm-bridge-dummy/src/main/resources/application-gate-auth.yml

```diff
@@ -0,0 +1,45 @@
+################################################################################
+# Copyright (c) 2021,2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+################################################################################
+
+
+bpdm:
+  client:
+    gate:
+      oauth2-client-registration: bridge-client
+      security-enabled: true
+
+spring:
+  security:
+    oauth2:
+      client:
+        provider:
+          catena-keycloak-provider:
```

Comment:
This hear specifies a new connection to a oauth2 server named "catena-keycloak-provider". The problem with this is that also the pool-client has the same client name. Better call them separately. Here "gate-oauth2-provider" and in the pool-auth "pool-oauth2-provider". That way both clients don't need to be under the same oauth2 host but may be secured through different servers. (Think about connecting a dev pool with an int gate for testing purposes for example)